### PR TITLE
remove a test case from badOpcodes

### PR DESCRIPTION
**Pull request by jwasinger**
Was mistakenly including `PC` in the `badOpcodes` test.

### CODE REVIEW

File: BlockchainTests/GeneralStateTests/stBadOpcode/badOpcodes_d101g0v0.json [hunk 1]
```diff
@@ -95,11 +96,12 @@
     },
     "badOpcodes_d101g0v0_EIP158" : {
         "_info" : {
+            "binaryenVersion" : "BINARYEN_VERSION",
             "comment" : "",
```
Comment (winsvega) [inline BlockchainTests/GeneralStateTests/stBadOpcode/badOpcodes_d101g0v0.json:286]: Wtf is binaryen? @axic

Comment (jwasinger) [reply BlockchainTests/GeneralStateTests/stBadOpcode/badOpcodes_d101g0v0.json:286]: It's a wasm interpreter.  Only relevant for `stEwasmTests` but somehow made it into this repository by accident.

**Comment by winsvega** [comment]
@jwasinger  please refill the tests on latest aleth. (without Binaryen)

**Comment by jwasinger** [comment]
This was filled with the latest aleth.

**Comment by jwasinger** [comment]
Oh I see.  This binaryen issue was solved.  I'll refill the tests.

File: src/GeneralStateTestsFiller/stBadOpcode/badOpcodesFiller.json [hunk 2]
```diff
@@ -37,7 +24,7 @@
 						},
 						{
                 "indexes" : {
-                    "data" : [115],
+                    "data" : [114],
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stBadOpcode/badOpcodesFiller.json]: where is the expect section for all other data for network `>=Constantinople` ?
if there is no expect section, then there will be no test generated. 
(for example there will be no test for data 100 and network Constantinople)

Comment (jwasinger) [reply src/GeneralStateTestsFiller/stBadOpcode/badOpcodesFiller.json]: Right.

**Comment by jwasinger** [comment]
I force pushed to only include the state test filler until I figure out why test cases are not passing:

```
Test Case "stBadOpcode":
100%
/home/ewasm/projects/aleth/test/tools/libtesteth/ImportTest.cpp(565): error: in "GeneralStateTests/stBadOpcode": badOpcodes_d120g0v0_Homestead Compare States: 6295ee1b4f6dd65047762f924ecd367c17eabf8f address not expected to exist!
Constantinople data: 4 gas: 0 val: 0
/home/ewasm/projects/aleth/test/tools/libtesteth/ImportTest.cpp(565): error: in "GeneralStateTests/stBadOpcode": badOpcodes_d120g0v0_Homestead Compare States: 6295ee1b4f6dd65047762f924ecd367c17eabf8f address not expected to exist!
Constantinople data: 5 gas: 0 val: 0
/home/ewasm/projects/aleth/test/tools/libtesteth/ImportTest.cpp(565): error: in "GeneralStateTests/stBadOpcode": badOpcodes_d120g0v0_Homestead Compare States: 6295ee1b4f6dd65047762f924ecd367c17eabf8f address not expected to exist!
Constantinople data: 6 gas: 0 val: 0
/home/ewasm/projects/aleth/test/tools/libtesteth/ImportTest.cpp(565): error: in "GeneralStateTests/stBadOpcode": badOpcodes_d120g0v0_Homestead Compare States: 6295ee1b4f6dd65047762f924ecd367c17eabf8f address not expected to exist!
Constantinople data: 24 gas: 0 val: 0
/home/ewasm/projects/aleth/test/tools/libtesteth/ImportTest.cpp(565): error: in "GeneralStateTests/stBadOpcode": badOpcodes_d120g0v0_Homestead Compare States: 6295ee1b4f6dd65047762f924ecd367c17eabf8f address not expected to exist!
ConstantinopleFix data: 4 gas: 0 val: 0
/home/ewasm/projects/aleth/test/tools/libtesteth/ImportTest.cpp(565): error: in "GeneralStateTests/stBadOpcode": badOpcodes_d120g0v0_Homestead Compare States: 6295ee1b4f6dd65047762f924ecd367c17eabf8f address not expected to exist!
ConstantinopleFix data: 5 gas: 0 val: 0
/home/ewasm/projects/aleth/test/tools/libtesteth/ImportTest.cpp(565): error: in "GeneralStateTests/stBadOpcode": badOpcodes_d120g0v0_Homestead Compare States: 6295ee1b4f6dd65047762f924ecd367c17eabf8f address not expected to exist!
ConstantinopleFix data: 6 gas: 0 val: 0
/home/ewasm/projects/aleth/test/tools/libtesteth/ImportTest.cpp(565): error: in "GeneralStateTests/stBadOpcode": badOpcodes_d120g0v0_Homestead Compare States: 6295ee1b4f6dd65047762f924ecd367c17eabf8f address not expected to exist!
ConstantinopleFix data: 24 gas: 0 val: 0
```

**Comment by jwasinger** [comment]
test case 4:

```
[1] PUSH1 0x01
[3] PUSH1 0x01
[5] PUSH1 0x01
[7] PUSH1 0x01
[9] PUSH1 0x01
[11] PUSH1 0x01
[13] PUSH1 0x01
[15] PUSH1 0x01
[16] '1b'(Unknown Opcode)
[18] PUSH1 0x00
[20] PUSH1 0x00
[21] RETURN
```

test case 5:
```
[1] PUSH1 0x01
[3] PUSH1 0x01
[5] PUSH1 0x01
[7] PUSH1 0x01
[9] PUSH1 0x01
[11] PUSH1 0x01
[13] PUSH1 0x01
[15] PUSH1 0x01
[16] '1c'(Unknown Opcode)
[18] PUSH1 0x00
[20] PUSH1 0x00
[21] RETURN
```

Is aleth using these opcodes for prototyping?

**Comment by winsvega** [comment]
@chfast, is aleth recognize this opcodes as valid when it should not ?

**Comment by jwasinger** [comment]
Okay, these are bit shifting opcodes.

**Comment by jwasinger** [comment]
And another one that is being updated (removed from the list of invalid opcodes) is `EXTCODEHASH`.

**Comment by winsvega** [comment]
Ok

**Pull request merged**